src/app/public/modules/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -0,0 +1,402 @@
+import {
+  ComponentFixture,
+  TestBed,
+  fakeAsync,
+  tick
+} from '@angular/core/testing';
+
+import {
+  expect,
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  DateRangePickerTestComponent
+} from './fixtures/date-range-picker.component.fixture';
+
+import {
+  DateRangePickerTestModule
+} from './fixtures/date-range-picker.module.fixture';
+
+import {
+  SkyDateRangeCalculation
+} from './date-range-calculation';
+
+import {
+  SkyDateRangeCalculatorId
+} from './date-range-calculator-id';
+
+import {
+  SkyDateRangeCalculatorType
+} from './date-range-calculator-type';
+
+describe('Date range picker', function () {
+  let fixture: ComponentFixture<DateRangePickerTestComponent>;
+  let component: DateRangePickerTestComponent;
+
+  function detectChanges(): void {
```

Comment:
Should we name this a little more descriptively. I wouldn't know by this name when to use this compared to just calling `fixture.detectChanges` directly.

Reply:
Any suggestions? I use it when I want the fixture to detect changes...